doc/02-Installation.md

```diff
@@ -1,15 +1,24 @@
-# Installation
+<!-- {% if index %} -->
+# Installing Icinga Web Graphite Integration
 
-## Requirements
+The recommended way to install this Integration is to use prebuilt
+packages for all supported platforms from our official release repository.
+Obviously [Icinga Web](https://icinga.com/docs/icinga-web) itself
```

Comment:
Starting the sentence with `Obviously` sounds rather harsh to me. Please rephrase that.